x/ibc/02-client/types/tendermint/misbehaviour.go

```diff
@@ -3,10 +3,67 @@ package tendermint
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types/errors"
+
+	"github.com/tendermint/tendermint/crypto"
+	tmtypes "github.com/tendermint/tendermint/types"
 )
 
+// Evidence is a wrapper over tendermint's DuplicateVoteEvidence
```

Comment:
Note: Bez also added [`Hash() cmn.HexBytes`](https://github.com/cosmos/cosmos-sdk/pull/5240/files#diff-de09f477bac07ddcdbd0cf4ded89f02eR15) to the interface, which needs to be added to the set of functions from this type.